include/picolibrary/indicator.h

```diff
@@ -120,6 +121,58 @@ class Fixed_Intensity_Indicator_Concept {
     auto toggle() noexcept -> Result<Void, Error_Code>;
 };
 
+/**
+ * \brief GPIO output pin based fixed intensity indicator
+ *
+ * \tparam Output_Pin The type of output pin being adapted.
```

Comment:
I don't know that saying that an output pin is being adapted is accurate.

Consider changing this to something along the lines of:
```suggestion
 * \tparam Output_Pin The type of output pin used to manipulate the indicator.
```